src/config.ts

```diff
@@ -177,9 +206,9 @@ export interface MarkConfig {
 
   // ---------- Interpolation: Line / area ----------
   /**
-   * The line interpolation method to use. One of linear, step-before, step-after, basis, basis-open, basis-closed, bundle, cardinal, cardinal-open, cardinal-closed, monotone.
+   * The line interpolation method to use. One of linear, step-before, step-after, basis, basis-open, cardinal, cardinal-open, monotone.
    */
-  interpolate?: string;
+  interpolate?: LineInterpolation;
```

Comment:
Let's name this `Interpolate`?  Because there is no other interpolate in the schema, no need to make different name.